src/main/java/net/helpscout/api/model/thread/Chat.java

```diff
@@ -1,10 +1,5 @@
 package net.helpscout.api.model.thread;
 
-/**
- * Chat -
- *
- * @author briandame@gmail.com
- */
 public class Chat extends AbstractThread {
 
```

Comment:
this feels a bit _empty_ - if it servers only as a marker class I'd at least add a comment here